pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
Wouldn't this only affect neovim's python environment? Why does it add a link for `/bin/python3` if collisions are a concern here?